test/EventStore.Client.Tests.Common/FluentDocker/TestService.cs

```diff
@@ -10,10 +11,14 @@ public interface ITestService : IAsyncDisposable {
 	Task Start();
 	Task Stop();
 	Task Restart(TimeSpan delay);
-	
-	Task Restart() => Restart(TimeSpan.Zero);
-	
-	void ReportStatus();
+
+#if NET
```

Comment:
no need, we simply add the missing interface implementation to the service.

`public Task Restart() => Restart(TimeSpan.Zero);`